docs/modules/ROOT/pages/tmail-backend/configure/index.adoc

```diff
@@ -134,4 +134,11 @@ Specified to TMail backend, we can configure the following configurations in the
 | https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys[S3 access key id]
 | objectstorage.s3.secondary.secretKey
 | https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys[S3 access key secret]
-|===
\ No newline at end of file
+|===
+
+Besides that, we need to disable S3 metrics to have the secondary blob store working, as we are having S3 metrics key conflicts between two blob stores.
```

Comment:
Secondary blob store likely should use another metric name.

Can we at least create an issue to come and fix this?

Reply:
https://github.com/linagora/james-project/issues/5297